project/AppDependencies.scala

```diff
@@ -20,13 +20,13 @@ object AppDependencies {
     "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
     "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
     "wolfendale"                  %% "scalacheck-gen-regexp"    % "0.1.2",
-    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  % "0.74.0",
```

Comment:
same here

Reply:
No issues with `0.74.0` it's just `1.x.x` onwards we need to be careful with

Reply:
Rolled back version on recent commit